java/client/src/main/java/glide/api/models/commands/RangeOptions.java

```diff
@@ -292,7 +293,23 @@ public RangeByScore(
 
     public static String[] createZrangeArgs(
             String key, RangeQuery rangeQuery, boolean reverse, boolean withScores) {
-        String[] arguments = new String[] {key, rangeQuery.getStart(), rangeQuery.getEnd()};
+        String[] arguments =
```

Comment:
make those instance functions (not static)

Reply:
I would then have to initialize RangeOptions which I don't want to do.